src/ui/EditorFrameset.js

```diff
@@ -9,9 +9,8 @@ export type EditorFramesetProps  = {
   className: ?string,
   embedded: ?boolean,
   header: ?React.Element<any>,
-  header: ?React.Element<any>,
   height: ?(string | number),
-  toolbar: ?React.Element<any>,
+  renderToolbarInFrameBody?: boolean,
```

Comment:
How about this?

```
toolbarPlacement: 'header' | 'body' | null,
```

I'm asking cause we may need to support different placement (e.g. at footer) for the comment editor.